packages/algob/test/internal/core/config/config-loading.ts

```diff
@@ -169,4 +190,79 @@ describe("config loading", function () {
       ).catch((err) => console.log(err)); ;
     });
   });
+
+  describe("load kmd accounts", function () {
+    useFixtureProject("config-project");
+    useEnvironment();
+
+    const fakeKmd: Kmd = {} as Kmd; // eslint-disable-line @typescript-eslint/consistent-type-assertions
+    const kmdOp = new KMDOperatorMock(fakeKmd);
+    let net: NetworkConfig;
+
+    beforeEach(function () {
+      net = this.env.config.networks.kmdNet;
+    });
+
+    afterEach(function () {
+      kmdOp.resetAccounts();
+    });
+
+    it("should ignore if kmd config is not defined", async function () {
+      net.kmdCfg = undefined;
+      const result = await _loadKMDAccounts(net, kmdOp);
+      assert.isUndefined(result);
+    });
+
+    it("should not connect to kmd if config is invalid", async function () {
+      const invalidKmdCfg = net.kmdCfg as KmdCfg;
+      invalidKmdCfg.port = 123; // invalid port
+
+      expectBuilderErrorAsync(
+        () => _loadKMDAccounts(net, new KMDOperator(createKmdClient(invalidKmdCfg))),
+        ERRORS.KMD.CONNECTION
+      ).catch((err) => console.log(err)); ;
+    });
+
+    it("network accounts should take precedence over KMD accounts", async function () {
```

Comment:
+ should detect conflict of account names 

can you please add this one too ?

Reply:
I wanted to. But i wasn't able to assert `console.warn(..)`. Do you have any suggestion?

Reply:
https://medium.com/@the_teacher/how-to-test-console-output-console-log-console-warn-with-rtl-react-testing-library-and-jest-6df367736cf0
this maybe helpful

Reply:
I checked, but it adds other dependencies. Didn't wanted to do that just for a single test.

Reply:
checking for a better solution

Reply:
I checked, we have `sinon` installed already. we can use that

Reply:
ok cool! Thanks